src/cli/tc.rs

```diff
@@ -104,11 +108,35 @@ pub async fn init(file_path: &str) -> Result<()> {
 pub fn display_schema(blueprint: &Blueprint) {
   Fmt::display(Fmt::heading(&"GraphQL Schema:\n".to_string()));
   let sdl = blueprint.to_schema();
-  Fmt::display(print_schema::print_schema(sdl));
+  Fmt::display(format!("{}\n", print_schema::print_schema(sdl)));
 }
 
 fn display_config(config: &Config, n_plus_one_queries: bool) {
-  Fmt::display(Fmt::success(&"No errors found".to_string()));
   let seq = vec![Fmt::n_plus_one_data(n_plus_one_queries, config)];
   Fmt::display(Fmt::table(seq));
 }
+
+fn validate_operations(blueprint: &Blueprint, operations: Vec<String>) -> Result<()> {
+  tokio::runtime::Runtime::new()?.block_on(async {
+    let schema = blueprint.to_validation_schema();
+    let mut execution = vec![];
+
+    for op in operations.iter() {
+      let operation = tokio::fs::read_to_string(op)
+        .await
+        .map_err(|_| anyhow!("Cannot read operation {}", op))?;
+        let Response { errors, .. } = schema.execute(&operation).await;
+        execution.push((op, errors));
+    }
```

Comment:
Now that we're handling errors with `map_err`, we can change this loop to a map. The code may require some tweaks to work 

```suggestion
    let execution = operations
      .iter()
      .map(async |op| {
        let operation = tokio::fs::read_to_string(op)
          .await
          .map_err(|_| anyhow!("Cannot read operation {}", op))? //This can likely be a CLIError
          
        let Response { errors, .. } = schema.execute(&operation).await;
        (op, errors)
      });
```